### ISSUE TITLE
Add non-AS2 documents

### ISSUE THREAD

**Issue by akuckartz** [jasnell/as2corpus#1]
Non-AS2 documents also can be helpful for tests. They should not violate JSON or JSON-LD syntax but only specific AS2 requirements.

**Comment by jasnell** [comment]
Yes, I would definitely like to include a directory full of non-valid AS2 examples that are expected to either fail or generate warnings
